_maps/RandomRuins/SpaceRuins/spacehotel.dmm

```diff
@@ -2081,6 +2379,17 @@
 /obj/structure/cable,
 /turf/open/floor/plating,
 /area/ruin/space/has_grav/hotel/power)
+"uQ" = (
+/obj/effect/mob_spawn/corpse/human/nanotrasenelitesoldier,
```

Comment:
thats a corpse with a lot of loot, and a centcom ID

Reply:
oops™